### PR TITLE
Add OCM commands migration step and misc script fixes

### PR DESCRIPTION
**Pull request by mikeshng**

### CODE REVIEW

File: dr-script/migrate-hcp.sh [hunk 1]
```diff
@@ -402,6 +402,25 @@ function restore_svc() {
...
+    fi
+
+    ocm patch /api/clusters_mgmt/v1/clusters/${HC_CLUSTER_ID}/hypershift <<-EOF
+{
+"management_cluster":"${MGMT2_CLUSTER_NAME}"
+}
+EOF
+    if [ $? -ne 0 ]; then
+        echo "Failed to patch OCM with new management cluster. Ensure you have the HCPMigration role."
+        exit 1
```
Comment (philipwu08) [inline dr-script/migrate-hcp.sh:420]: Mike, do we want to log and continue given where we are in the script. Basically, the HC is migrated and the teardown is remaining

File: dr-script/migrate-hcp.sh [hunk 2]
```diff
@@ -612,6 +633,14 @@ echo "Restoration Done!"
 ELAPSED="Elapsed: $(($SECONDS / 3600))hrs $((($SECONDS / 60) % 60))min $(($SECONDS % 60))sec"
 echo $ELAPSED
 
+## OCM Migration commands
+SECONDS=0
+echo "Executing OCM migration commands"
+ocm_migration
+echo "OCM migration commands Done!"
+ELAPSED="Elapsed: $(($SECONDS / 3600))hrs $((($SECONDS / 60) % 60))min $(($SECONDS % 60))sec"
```
Comment (philipwu08) [inline dr-script/migrate-hcp.sh:641]: Maybe we could remove the timing stuff for the ocm_migration. It should take at most 1 second

Comment (mikeshng) [reply dr-script/migrate-hcp.sh]: I agree. Thanks. Done as suggested.

File: dr-script/migrate-hcp.sh (hunk 1 above)
Comment (mikeshng) [reply dr-script/migrate-hcp.sh:420]: move to the end for now.

**Comment by mikeshng** [comment]
Merging as agreed to by Phil.

**Pull request merged**